docs/CodeStyle.md

```diff
@@ -32,6 +32,13 @@
 ## C++
   * used only for unit testing
   * lower-case class names (same as stl/boost)
+  * the tests are written using [C++11](https://en.cppreference.com/w/cpp/11). Whenever applicable, the usage of advanced language features is allowed and preferred over legacy code. For example:
+    * Prefer references over pointers
+    * `auto` for type deduction 
+    * Use [move semantics](https://www.cprogramming.com/c++11/rvalue-references-and-move-semantics-in-c++11.html) where applicable
+    * `constexpr` for compile-time values, instead of `const` or `#define`
+    * `using` instead of `typedef`
+    * List initialization
```

Comment:
maybe "Initializer list"?
or it could be interpreted incorrectly, e.g. `std::list` initialization

Reply:
I've add:
```
* range-based `for` instead of `for` over iterators
```

Reply:
"List initialization" is the formal name of the feature:
https://en.cppreference.com/w/cpp/language/list_initialization

Reply:
> "List initialization" is the formal name of the feature:
https://en.cppreference.com/w/cpp/language/list_initialization

ah, ok. then better use official naming. thanks!